src/blackconfig/buildconfig_gen.inc.in

```diff
@@ -65,6 +65,15 @@ constexpr bool BlackConfig::CBuildConfig::isCompiledWithFsxSupport()
 !!ENDIF
 }
 
+constexpr bool BlackConfig::CBuildConfig::isCompiledWithMsFlightSimulatorSupport()
```

Comment:
Ah, that makes sense. I just assumed it was for MSFS because every other flight sim had the same sort of function. I'll get that corrected, including the docs. I haven't touched C++ since C++14 and Rust doesn't have C's declaration/implementation split. so looking for docs in the header didn't occur to me, my apologies for that.

Do you have any objections to using "isCompiledWithMsfsSupport" to keep the capitalization consistent as well?

Reply:
`isCompiledWithMsfsSupport` would be fine with me.